### ISSUE TITLE
Refactor routes

### ISSUE THREAD

**Issue by andycochran** [NYCPlanning/labs-applicantmaps#200]
- `project/new` -- remove the geometry creation, just have a form
- `project/:id` -- conditional content, prompting questions
- new routes: 
  - `project/:id/geometry-edit?type=[type],mode=[lot,draw]`
  - types:  `development-site`
    - `project-area`
    - `underlying-zoning`
    - `commercial-overlays`
    - `special-purpose-districts`
    - `rezoning-area`

**Comment by hannahkates** [comment]
I've created the new routes (minus the query params for projects/:id/geometry-edit), but I'm not sure about the best way to now approach linking them all. I think we had talked about having the controller for projects/:id handle all the logic that determines where a button click should take you.

**Comment by allthesignals** [comment]
What are the required fields at each point in the process? We should have a `stepsCompleted` computed and a sense of which fields are required at each step.

**Comment by allthesignals** [comment]
I'm thinking about form validation - when are "complete" buttons for forms disabled? We have model validation, but also need that validation to be split out into steps.

**Comment by hannahkates** [comment]
I was thinking we could determine where the user is in the process based on a series a boolean inputs (ex: do you need rezoning?) and checking if they have created the geometry they say they need for all the required boolean answers. Along the lines of the properties below -- but there maybe a simpler way to do it.

```
  @attr('boolean', { defaultValue: null }) needProjectArea;

  @attr('boolean', { defaultValue: null }) needRezoning;

  @attr('boolean', { defaultValue: null }) needUnderlyingZoning;

  @attr('boolean', { defaultValue: null }) needCommercialOverlay;

  @attr('boolean', { defaultValue: null }) needSpecialDistrict;

  @computed('needProjectArea', 'needRezoning', 'needUnderlyingZoning', 'needCommercialOverlay', 'needSpecialDistrict')
  get isComplete() {
    // if they inputted true and the corresponding geom exists in the model, return true
    // else if they inputted false (i.e don't have rezoning), return true
    // else if any of the need___ booleans are null, return false because they haven't answered a required question
    // else false
  }
  }
```

**Comment by allthesignals** [comment]
@hannahkates makes sense - those booleans need to be explicit fields in the database. We should just be sure to comment stuff so fields are grouped nicely.
